flake.nix

```
@@ -2,23 +2,32 @@
   inputs = {
     crane.url = "github:ipetkov/crane";
     utils.url = "github:numtide/flake-utils";
+    nixpkgs.follows = "crane/nixpkgs";
   };
 
   outputs = {
     self,
     crane,
     utils,
+    nixpkgs,
     ...
-  }: let
-    supportedSystems = ["x86_64-linux" "x86_64-darwin" "aarch64-linux"];
-  in
+  }:
+    let
+      supportedSystems = ["x86_64-linux" "x86_64-darwin" "aarch64-linux"];
+    in
     utils.lib.eachSystem supportedSystems
     (
-      system: {
-        packages.oura = crane.lib.${system}.buildPackage {
-          src = self;
-        };
-        packages.default = self.packages.${system}.oura;
-      }
+      system:
+        let
+          pkgs = import nixpkgs { inherit system; };
+        in
+        {
```

this is now not needed
```suggestion
      system: {
```